docs/DeveloperGuide.md

```diff
@@ -138,41 +138,43 @@ method call to return `commandHistory`'s 2nd last command instead of the last co
 With `addToHistory(String command)`, `previousCommand()`, `nextCommand()` and `currentCommand()` implemented, a simple `setOnKeyPressed` under `CommandBox` class which checks
 for user's input of arrow up (which calls previousCommand()) and arrow down (which calls nextCommand()) would suffice for GUI implementation.
 
-### \[Proposed\] Undo/redo feature
+### Undo/Redo Command
 
-#### Proposed Implementation
+Each `Model` internally stores its undo and redo history as a (for `DeliveryModel`) `deliveryBookStateList` and `deliveryBookStatePointer`. There are corresponding analogs for `InventoryModel`.
+Additionally, the following commands are implemented by `ModelsManager`.
 
-The proposed undo/redo mechanism is facilitated by `VersionedAddressBook`. It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+* `ModelsManager#commit()` — Saves the current book states of all the `Model`s it contains in their history.
+* `VersionedAddressBook#undo()` — Restores the previous book states from each `Model` from their history.
+* `VersionedAddressBook#redo()` — Restores a all previously undone book states from every `Model`'s history.
 
-* `VersionedAddressBook#commit()` — Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` — Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` — Restores a previously undone address book state from its history.
+These operations are exposed in the `Models` interface as `Models#commit()`, `Models#commit()` and `Models#redo()` respectively.
```

Comment:
You mean Models#undo for the second one?

Reply:
Yes, good catch